pages/docs/reference/classes.md

```diff
@@ -261,6 +261,21 @@ class Example // Implicitly inherits from Any
 
 `Any` has three methods: `equals()`, `hashCode()` and `toString()`. Thus, they are defined for all Kotlin classes. 
 
+#### "open" Keyword
+In Kotlin, all the classes are final by default i.e. they can’t be inherited.
+To make it inheritable, you must mark it with the keyword **open**. Otherwise it will throw error saying “type is final so can’t be inherited”. 
+ *This is opposite to Java where you have to make your class final explicitly.*
```

Comment:
Please remove this.
We deliberately avoid comparisons with Java in Kotlin docs, so that people with no Java background understand them better.